simulator/simple_planning_simulator/README.md

```diff
@@ -0,0 +1,62 @@
+# Simple Planning Simulator
+This node simulates the vehicle motion for a vehicle command in 2D using a simple vehicle model.
+
+## Interface
+
+**input**
+ - /control/vehicle_cmd [`autoware_vehicle_msgs/VehicleCommand`] : target command to drive a vehicle.
+ - /initialpose [`geometry_msgs/PoseWithCovarianceStamped`] : for initial pose
+ - /initialtwist [`geometry_msgs/TwistStamped`] : for initial velocity
+ - /planning/scenario_planning/trajectory [`autoware_planning_msgs/Trajectory`]: for z position
+ - /vehicle/engage
```

Comment:
/vehicle/engage
send true: the simulator (re)starts.
send false: the simulator pauses.